Makefile

```diff
@@ -5,47 +5,12 @@ all: install test
 
 install:
 	# Install all dependencies
-	$(POETRY) install -E all
-	pip install flash-attn
-
-install-qr-reader:
-	# Install qr-reader dependencies only
 	sudo apt update
 	sudo apt-get install -y libzbar0
-	$(POETRY) install -E qr-reader --no-interaction
-
-install-owlv2:
-	# Install owlv2 dependencies only
-	$(POETRY) install -E owlv2 --no-interaction
-
-install-zeroshot-counting:
-	# Install loca dependencies only
-	$(POETRY) install -E loca-model
-
-install-depth-anything-v2:
-	# Install depth-anything-v2 dependencies only
-	$(POETRY) install -E depth-anything-v2-model --no-interaction
-
-install-florencev2:
-	# Install florencev2 dependencies only
-	$(POETRY) install -E florencev2 --no-interaction
-	pip install flash-attn
-
-install-nsfw-classification:
-	# Install nsfw_classification dependencies only
-	$(POETRY) install -E nsfw-classification --no-interaction
-
-install-controlnet-aux:
-	# Install controlnet_aux dependencies only
-	$(POETRY) install -E controlnet-aux --no-interaction
-
-install-florencev2-qa:
-	# Install florencev2-qa dependencies only
-	$(POETRY) install -E florencev2-qa --no-interaction
-	pip install flash-attn
```

Comment:
@camiloaz do you think that it makes sense to install all tools at once instead of allowing users to choose what model to install?

Reply:
people can still install specific models, just not with `make` commands. doing this just adds too much things to do for every tool that we add and it's not necessary.